smile_id_core/constants.py

```diff
@@ -1,6 +1,9 @@
 class Servers:
```

Comment:
Would propose to do something like this:
```python
class Servers:
    TEST_SERVER_URL = "https://testapi.smileidentity.com"
    LIVE_SERVER_URL = "https://api.smileidentity.com"
```
and then in the api class something like
```python
class WebApiTest(ApiBase):
    SERVER_URL = Servers.TEST_SERVER_URL

    class Urls:
        GET_JOB_STATUS = "{server_url}/v{api_version}/job_status"
        GET_SERVICES = "{server_url}/v{api_version}/services"
        UPLOAD = "{server_url}/v{api_version}/upload"
        VERIFY_DOCUMENT = "{server_url}/v{api_version}/id_verification"
        VERIFY_DOCUMENT_ASYNC = "{server_url}/v{api_version}/verify_async"

    def __init__(self, partner_id: str, api_key: str, api_version: int = 1, call_back_url: str = None):
        ...
        self.api_version = api_version
```
Reasons:
- Eliminating variable names `V2`
- Having only 2 urls for test and prod servers